Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -903,6 +896,45 @@ void AODProducerWorkflowDPL::init(InitContext& ic)
   // Needed by MCH track extrapolation
   o2::base::GeometryManager::loadGeometry();
 
+  // writing metadata if it's not yet in AOD file
+  // note: `--aod-writer-resmode "UPDATE"` have to be used,
+  //       so that metadata is not overwritten
+  mResFile += ".root";
+  auto* fResFile = TFile::Open(mResFile, "UPDATE");
```

Comment:
Shouldn't this be written by using some O2 framework functions instead of doing it manually?

Reply:
It would be nice, but I'm not aware of a method that would allow writing an object outside of DF folder. Also, the code in this PR got mixed with some older code, so I will make another PR, where we can continue discussion :)

Reply:
@ktf could you comment?